apps/omg_status/lib/omg_status/metric/event.ex

```diff
@@ -104,6 +110,7 @@ defmodule OMG.Status.Metric.Event do
   defp events_name(:piggyback_challenges_processor), do: "piggyback_challenges_processor_ethereum_events"
   defp events_name(:ife_exit_finalizer), do: "ife_exit_finalizer_ethereum_events"
 
+  defp message_queue_len_name(:block_queue), do: "block_queue_message_queue_len"
```

Comment:
What's that `len` thing?

Reply:
number of messages in the process message queue ```http://erlang.org/doc/man/erlang.html#process_info-2```
```
{message_queue_len, MessageQueueLen :: integer() >= 0}
```